unit_tests/test_decode_backtrace.py

```diff
@@ -190,3 +195,142 @@ def test_04_decode_backtraces_core(self):
             if event.get('backtrace') and event.get('raw_backtrace'):
                 self.assertEqual(event['backtrace'].strip(),
                                  "addr2line -Cpife scylla_debug_info_file {}".format(' '.join(event['raw_backtrace'].split("\n"))))
+
+
+# Pytest-style fixtures and tests
+
+@pytest.fixture
+def test_config_fixture(events_function_scope):
```

Comment:
why do we need `events_function_scope` first?

Reply:
we don't